src/riscv_private.h

```diff
@@ -130,6 +116,21 @@ struct riscv_internal {
     uint32_t csr_mbadaddr;
 
     bool compressed; /**< current instruction is compressed or not */
+
+#if RV32_HAS(GDBSTUB)
```

Comment:
Why did you make such change?

Reply:
Decrease the definition in gencode, if we put the fields needed by jit at the top of the data structure, or we need to define datastructure about GDB in gencode.

Reply:
Explain in git commit messages.